code/modules/mob/living/simple_animal/hostile/true_changeling.dm

```diff
@@ -0,0 +1,215 @@
+#define TRUE_CHANGELING_REFORM_THRESHOLD 0 //Can turn back at will, by default
+#define TRUE_CHANGELING_PASSIVE_HEAL 3 //Amount of brute damage restored per tick
+#define TRUE_CHANGELING_FORCED_REFORM 180 //3 minutes
+
+//Changelings in their true form.
+//Massive health and damage, but all of their chems and it's really obvious it's >them
+
+/mob/living/simple_animal/hostile/true_changeling
+	name = "ancient horror"
+	real_name = "ancient horror"
+	desc = "Holy shit, what the fuck is that thing?!"
+	speak_emote = list("says with one of its faces")
+	emote_hear = list("says with one of its faces")
+	icon = 'icons/mob/changeling.dmi'
+	icon_state = "horror1"
+	icon_living = "horror1"
+	icon_dead = "horror_dead"
+	speed = -0.5
```

Comment:
horrible slowness is the entire reason horror form was always a really terrible option as opposed to just running around with all of your sweet ling abilities, bring horror form up to speed and make it the true final form it's meant to be

Reply:
i did, it's original speed was 3.

Reply:
i don't know normal speed but i think it's -1 for a comparison, 3 is like jugger speed